.circleci/config.yml

```diff
@@ -8,28 +8,43 @@ commands:
       - run:
           name: Version Information
           command: rustc --version; cargo --version; rustup --version
-      - run:
-          name: Install Dependencies
-          command: |
-            sudo sh -c 'echo "deb http://deb.debian.org/debian stretch-backports main" > /etc/apt/sources.list.d/backports.list'
-            sudo apt-get update
-            sudo apt-get install -y protobuf-compiler/stretch-backports cmake golang curl
-            sudo apt-get clean
-            sudo rm -r /var/lib/apt/lists/*
-            rustup component add clippy rustfmt
       - run:
           name: Setup Env
           command: |
             echo 'export TAG=0.1.${CIRCLE_BUILD_NUM}' >> $BASH_ENV
             echo 'export IMAGE_NAME=myapp' >> $BASH_ENV
+            echo 'export CARGO_INCREMENTAL=0' >> $BASH_ENV
+            echo 'export SCCACHE_CACHE_SIZE=2G' >> $BASH_ENV
+            echo 'export RUSTC_WRAPPER=sccache' >> $BASH_ENV
+            echo 'export CC="sccache cc"' >> $BASH_ENV
+            echo 'export CXX="sccache c++"' >> $BASH_ENV
+  restore_sccache_cache:
+    steps:
+      - restore_cache:
+          name: Restore sccache Cache
+          key: sccache-cache-{{ arch }}-{{ .Environment.CIRCLE_JOB }}
+  save_sccache_cache:
+    steps:
+      - run:
+          name: Show sccache Stats
+          command: sccache -s
+      - save_cache:
+          name: Save sccache Cache
+          key: sccache-cache-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ epoch }}
+          paths:
+            - "~/.cache/sccache"
 
 jobs:
   build:
     docker:
-      - image: circleci/rust:stretch
+      - image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/github_circleci:latest
```

Comment:
Should we hardcode this? Wouldn't `ecr.us...` resolve to the same?
https://docs.aws.amazon.com/general/latest/gr/rande.html#ecr_region

Reply:
Ah, no. 
Can we get the output of `aws ecr get-login --region us-west-2 --registry-ids <foo> --no-include-email` in here through an env var maybe?

Reply:
The docker login token you get is only valid for 12 hours according to https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login.html:
"This command retrieves a token that is valid for a specified registry for 12 hours, and then it prints a docker login command with that authorization token."